en_US.json

```diff
@@ -749,6 +749,26 @@
 	"edgeHostname.update.success.modal.content": "The Edge Hostname has been successfully updated.",
 	"edgeHostname.update.success.modal.title": "Edge Hostname Updated",
 	"edgeHostname.validation.error.desc.too.long": "The description can be up to 250 characters.",
+	"edgeHostname.zone.action.type.deliver.by.type1.ipv4": "Deliver by all server groups including the ones in China, with IPv4 only.",
+	"edgeHostname.zone.action.type.deliver.by.type1.ipv6": "Deliver by all server groups including the ones in China, with IPv4 and v6.",
```

Comment:
I think it's better to say "IPv6" instead of "v6"

Reply:
Thanks @burtonquantil .
I changed the "v6" to "IPv6"